lib/model.js

```diff
@@ -2642,7 +2642,7 @@ class Model {
               const associationInstance = createdAssociationInstances[idx];
               const instance = associationInstanceIndexToInstanceMap[idx];
 
-              instance[include.association.accessors.set](associationInstance, { save: false, logging: options.logging });
+              await include.association.set(instance, associationInstance, { save: false, logging: options.logging });
```

Comment:
`save: false`, means this will not return promise. So await is not useful here. Why `accessors` is being removed?

Reply:
All of the `set` function were converted to async so they *always* return a promise now.
Using `accessors.set` was only used to look up the instance `set` method which is effectively just a proxy (see helpers.js:64) seemed redundant to me 🤷‍♂️

Reply:
Ah yes, it is an async method now. That make sense